docs/_instructions/buildtool.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+class: bnd
+title: -buildtool  toolspec  (EXPERIMENTAL!)
```

Comment:
How does this related to the templates in Bndtools which setup a new workspace? Should they be kept in sync/use the same source repo?

Reply:
first see how this works out, I am tired of all the time having to copy these gradle files. So I am trying to create a clean workspace and then use this tool to install gradle (or another tool) with a Github action. If it works well, we should take a good look at our workspaces and reduce. but one step at a time.